tests/sentry/replays/test_organization_replay_index.py

```diff
@@ -307,3 +307,229 @@ def test_get_replays_pagination(self):
             response_data = response.json()
             assert "data" in response_data
             assert len(response_data["data"]) == 0
+
+    def test_get_replays_user_filters(self):
```

Comment:
could we add a few tests for bad queries? e.g. incorrect operator, bad type, etc.

Reply:
👍